include/ceed/jit-tools.h

```diff
@@ -0,0 +1,26 @@
+/// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
+/// the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
+/// reserved. See files LICENSE and NOTICE for details.
+///
+/// This file is part of CEED, a collection of benchmarks, miniapps, software
+/// libraries and APIs for efficient high-order finite element and spectral
+/// element discretizations for exascale applications. For more information and
+/// source code availability see http://github.com/ceed.
+///
+/// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
+/// a collaborative effort of two U.S. Department of Energy organizations (Office
+/// of Science and the National Nuclear Security Administration) responsible for
+/// the planning and preparation of a capable exascale ecosystem, including
+/// software, applications, hardware, advanced system engineering and early
+/// testbed platforms, in support of the nation's exascale computing imperative.
+
+#ifndef _ceed_jit_h
+#define _ceed_jit_h
+
+#include <ceed/ceed.h>
+
+#define CEED_MAX_PATH_LEN 1024
```

Comment:
This is never used? It'd be best if we can avoid needing to hard-code such a thing. There is `PATH_MAX` and `pathconf` that can be used, but see this BUGS section.
https://www.man7.org/linux/man-pages/man3/realpath.3.html#BUGS

Reply:
Good catch - holdover from old approach

Reply:
fixed